apis/ecx/v1alpha1/zz_l2connectionaccepter_types.go

```diff
@@ -35,18 +37,22 @@ type L2ConnectionAccepterObservation struct {
 
 type L2ConnectionAccepterParameters struct {
 
+	// and secret_key resource arguments
```

Comment:
This line of generated text appears to be coming from the resource docs overview rather than from the attributes block.